workers.py

```diff
@@ -88,6 +139,9 @@ def worker() -> None:
             sample_size,
         )
         result = isolation_forest.predict(data_frame)
+        scores = compile_scores(result)
```

Comment:
@jameswnl Can you please move these 3 variables (`scores`, `contrasts` and `charts`) directly into the output JSON? It's gonna remove 3 variables from this function and make pylama happy, so travis would be passing.

Reply:
done